.github/ISSUE_TEMPLATE/pipeline-issue-report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Pipeline issue report
+about: Report a potential issue related to the pipeline editor or pipeline execution
+title: ''
+labels: component:pipeline-editor, component:pipeline-runtime, status:Needs Triage
+assignees: ''
+
+---
+
+**Describe the issue**
+A clear and concise description of what the issue is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Screenshots or log output**
+If applicable, add screenshots or log output to help explain your problem.
```

Comment:
This could also be helpful for long outputs
```suggestion
If applicable, add screenshots or log output to help explain your problem.
<details><summary>Log Output</summary>
<pre>
Paste the log output here.
</pre>
</details>
```